opentreemap/treemap/js/src/diameterCalculator.js

```diff
@@ -21,14 +21,14 @@ function elementToValue(el) {
     return $(el).data('value');
 }
 
-function zeroToEmptyString(n) {
-    return n === 0 ? '' : n;
-}
-
 function textToFloat(t) {
     var f = parseFloat(t);
 
-    return isNaN(f) ? 0 : f;
+    return isValidNumber(f) ? f : 0;
+}
+
+function isValidNumber(num) {
```

Comment:
because we used Ramda, if desired, you could probably convert this to:
`var isValidNumber = R.and(_.isNumber, R.not(isNan));`